python/src/tasks/cluster_cells.py

```diff
@@ -20,6 +20,7 @@ def _format_result(self, raw, cell_set_key, cell_set_name):
             "key": cell_set_key,
             "name": cell_set_name,
             "rootNode": True,
+            "type": "cellSets",
```

Comment:
is there a need to add a `type` property here? Will it be used later in other parts of worker / UI?

Reply:
yes it is expected and used in different parts of the ui, for example `scratchpad` and `louvain` are type 'cellSets', while `sample` is type 'metadataCategorical'. If we don't set it, then the received data will still be loaded with a `type` property but it will be undefined.